### ISSUE TITLE
Sección Recetas

### ISSUE THREAD

**Issue by ffosset** [soydetemporada/website#1]
*Crear base de datos con siguientes columnas*
1. ID
2. Name
3. Category
4. Ingredient 1
5. Ingredient 2
6. Ingredient 3
7. Source Name
8- Source URL
9- Image

- Diseñar la página

**Comment by ffosset** [comment]
For more accuracy in the recipe results, I excluded "patata" "ajo" and "cebolla" in the ingredients list as they are stored after harvest for the whole year.

**Comment by raiabril** [comment]
Yep! To do this, we can extract for each month and each product, the list of other products available, then we can assign the featured recipes per product-month.
